shared/src/store/action-reducers/patient.ts

```diff
@@ -56,7 +57,14 @@ export class SetVisibleStatusAction implements Action {
     @IsUUID(4, uuidValidationOptions)
     public readonly patientId!: UUID;
 
-    @IsString()
+    @IsStringLiteralUnion<PatientStatus>({
+        black: true,
+        blue: true,
+        green: true,
+        red: true,
+        white: true,
+        yellow: true,
+    })
```

Comment:
Maybe define the allowed values object where you define the type (as a convention)? This would keep the code short, make refactoring easier, and remove duplication.

so in 

````ts
export type PatientStatus = keyof typeof patientStatusAllowedLiterals;

// TODO: In the best case, the name should be self-explanatory, and we wouldn't need a comment - Can you come up with a better name?

// Mostly used for validation
export const allowedPatientStatusLiterals = {
    black: true,
    blue: true,
    green: true,
    red: true,
    white: true,
    yellow: true,
};
````

Reply:
Yes, this would be the plan. I'll refactor this.